client/signup/steps/new-or-existing-site/index.tsx

```diff
@@ -18,8 +21,10 @@ export default function NewOrExistingSiteStep( props: Props ): React.ReactNode {
 	const dispatch = useDispatch();
 	const translate = useTranslate();
 
-	const headerText = translate( 'Choose where you want us to build your site.' );
-	const subHeaderText = '';
+	const headerText = translate( 'Do it For Me' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Do It For Me (Lite)' )` *ES Score: 7*